romancal/tweakreg/tweakreg_step.py

```diff
@@ -167,166 +151,174 @@ def process(self, input):
             raise ValueError("Input must contain at least one image model.")
 
         # Build the catalogs for input images
-        for i, image_model in enumerate(images):
-            if image_model.meta.exposure.type != "WFI_IMAGE":
-                # Check to see if attempt to run tweakreg on non-Image data
-                self.log.info("Skipping TweakReg for spectral exposure.")
-                # Uncomment below once rad & input data have the cal_step tweakreg
-                # image_model.meta.cal_step.tweakreg = "SKIPPED"
-                return image_model
-
-            if hasattr(image_model.meta, "source_detection"):
-                is_tweakreg_catalog_present = hasattr(
-                    image_model.meta.source_detection, "tweakreg_catalog"
-                )
-                is_tweakreg_catalog_name_present = hasattr(
-                    image_model.meta.source_detection, "tweakreg_catalog_name"
-                )
-                if is_tweakreg_catalog_present:
-                    # read catalog from structured array
-                    catalog = Table(
-                        np.asarray(image_model.meta.source_detection.tweakreg_catalog)
+        with images:
+            for i, image_model in enumerate(images):
+                if image_model.meta.exposure.type != "WFI_IMAGE":
+                    # Check to see if attempt to run tweakreg on non-Image data
+                    self.log.info("Skipping TweakReg for spectral exposure.")
+                    # Uncomment below once rad & input data have the cal_step tweakreg
+                    # image_model.meta.cal_step.tweakreg = "SKIPPED"
```

Comment:
@mairanteodoro , this can probably be uncommented now?

Reply:
Resolving due to lack of response.

Reply:
> @mairanteodoro , this can probably be uncommented now?

Done in [PR#1276](https://github.com/spacetelescope/romancal/pull/1276).